zsh/aliases.zsh

```diff
@@ -18,6 +18,9 @@ fi
 # Just bcoz clr shorter than clear
 alias clr='clear'
 
+# Go to the /home/$USER (~) directory and clears window of your terminal
```

Comment:
Better to say `~`, because on macOS we have `/Users/$USER`. I'm gonna fix that be myself